doc/manual/install-judgehost.rst

```diff
@@ -53,12 +53,13 @@ home directory::
 For running solution programs under a non-privileged user, a user and group have
 to be added to the system that acts as judgehost. This user does not
 need a home-directory or password, so the following command would
-suffice to add a user and group ``domjudge-run-1`` with minimal privileges::
+suffice to add a user and group ``domjudge-run-2`` with minimal privileges
+with the judgedaemon restricted to CPU core 2::
```

Comment:
Do we need to say somehwere that you need a machine with at least 3 cores to run a judgehost?

Reply:
I think I always assume someone has more than 3 cores but the 2 is now chosen because 1 might be the `hyperthreading` core which is not clearly explained, the person who wouldn't have hyperthreading should have a CPU(1) and if you run a single core machine I think you're aware of what you're doing?

Reply:
I think it's not uncommon in a virtualised environment to have 'only' 1 or 2 cores right?

Reply:
I did not think of that, and that is indeed a "normal" usecase.